core/src/main/scala/kafka/server/ApiVersionManager.scala

```diff
@@ -112,7 +107,7 @@ class SimpleApiVersionManager(
       setSupportedFeatures(brokerFeatures).
       setFinalizedFeatures(currentFeatures.finalizedFeatures()).
       setFinalizedFeaturesEpoch(currentFeatures.finalizedFeaturesEpoch()).
-      setZkMigrationEnabled(zkMigrationEnabled).
+      setZkMigrationEnabled(false).
```

Comment:
why not removing the `setZkMigrationEnabled`?

Reply:
We cannot remove `setZkMigrationEnabled` from the RPC without breaking backwards compatibility. Although it does default to `false`, so we could leave out this line (which I think might be more confusing to read, but would be technically correct).

Reply:
Maybe we can hardcode `zkMigrationReady` to `false` when building `ApiVersionsResponse` (`ApiVersionsResponse#Builder#build`) and then remove this setter ?

Reply:
Wouldn't that make testing harder? It seems fine to just set it here.